Configuration/src/main/java/io/deephaven/configuration/Configuration.java

```diff
@@ -22,7 +22,7 @@
  * configuration pieces.
  */
 @SuppressWarnings({"WeakerAccess", "unused"})
-public class Configuration extends PropertyFile {
+public abstract class Configuration extends PropertyFile {
     private static final String DEFAULT_CONF_NAME = "default";
```

Comment:
I think we should delete this constant. Users can introduce a "default" named configuration, but this should be distinct from the default configuration. The current code is more complicated than it needs to be, because it assumes users will want to ask for "default" by name. We can avoid one possible source of confusion by completely separating the two.